frame/lending/src/tests.rs

```diff
@@ -1578,24 +1591,38 @@ fn create_simple_market() -> (MarketIndex, VaultId) {
 ///
 /// Panics on any errors and checks that the last event was `CollateralDeposited` with the correct/
 /// expected values.
-fn mint_and_deposit_collateral<T: crate::Config>(
-	account: &sp_core::sr25519::Public,
+pub fn mint_and_deposit_collateral<R>(
```

Comment:
Again, use `T` for the config type

Reply:
(Or something more verbose like `Runtime`)

Reply:
Done